lib/mixlib/install/product.rb

```diff
@@ -193,9 +193,33 @@ def lookup(key, version = :latest)
 
   product "automate" do
```

Comment:
Treat the `automate` and `delivery` products as separate entities. No backwards compat logic for which product name they specify. That will remove the copy/paste issue here and logic that is needed here like the ctl name change will be more clear.